eks-anywhere-common/Addons/Partner/Pulumi/pulumi-tester-role.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
If this configmap is for tester, this should be moved to `Testers/Pulumi` folder.